apps/flame-defi/app/components/deposit-card/deposit-card.tsx

```diff
@@ -525,12 +525,9 @@ export default function DepositCard(): React.ReactElement {
       </div>
 
       <div className="mt-4">
-        <ActionButton
-          callback={handleDeposit}
-          disabled={isDepositDisabled}
-          isLoading={isLoading}
-          buttonText={"Deposit"}
-        />
+        <Button onClick={handleDeposit} disabled={isDepositDisabled}>
```

Comment:
This makes the button style incorrect I think

<img width="659" alt="Screenshot 2025-03-18 at 4 31 04 PM" src="https://github.com/user-attachments/assets/e0eea9b5-4f10-41a7-b4ee-b8f4ace13c6e" />

Reply:
Whoops! I'm a goober and was wrong here. I should probably look at the correct designs before commenting on ui 😅